nixos/tests/systemd-machinectl.nix

```diff
@@ -1,4 +1,4 @@
-import ./make-test-python.nix ({ pkgs, ... }:
+import ./make-test-python.nix ({ pkgs, evalSystemConfiguration, ... }:
```

Comment:
in `make-test-python.nix`: `evalSystemConfiguration` this is not becoming a staple.

In general; explicit passing:

- pro: we know which tests evaluate a system config yet again
- con: it's a verbose calling convention and by making nixoslib extend lib, we could clean the iface and make it part of the global context.


eternal truth: global context is bad for the learning curve.